io.github.yesser_studios.Pong.yml

```diff
@@ -0,0 +1,55 @@
+id: io.github.yesser_studios.Pong
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+add-extensions:
+  org.freedesktop.Platform.GL.default:
+    version: '1.0'
+    directory: extension_directory
+    add-ld-path: lib
+    merge-dirs: plug-ins;help
+    subdirectories: true
+    no-autodownload: true
+    autodelete: true
+build-options:
+  prepend-path: "/usr/lib/sdk/dotnet8/bin"
+  append-ld-library-path: "/usr/lib/sdk/dotnet8/lib"
+  prepend-pkg-config-path: "/usr/lib/sdk/dotnet8/lib/pkgconfig"
+finish-args:
+  - --device=dri
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+
+command: Pong.Desktop
+modules:
+  - name: dotnet
+    buildsystem: simple
+    build-commands:
+    - /usr/lib/sdk/dotnet8/bin/install.sh
+
+  - name: pong
+    buildsystem: simple
+    build-options:
+      build-args:
+        - --share=network
```

Comment:
build-args are not allow and build MUST be offline.

Reply:
This is an issue because even after following the details on how to generate a nuget package source files, dotnet publish still crashes on tool restoration. Is there a way to do something similiar for tools?

Reply:
I resolved the issue by including a pre-built binary to download from GitHub.